### DISCUSSION TITLE
Ray doesn't start on windows 10

### DISCUSSION THREAD

**Discussion by pbruelemans** [spatie/ray#370] [Q&A] [answered]
I have been using a licensed version of Ray for a few weeks now, without any problems.  Since yesterday I noticed that ray stopped showing debug messages.
I restarted ray and got the message to update it. I did, but now ray doesn't seem to work anymore.  
I tried reinstalling it (downloaded the installer again), rebooting, cleaning the folders \Appdata\local\ray and \appdata\roaming\ray... Nothing seems to work.
When I start ray, nothing seems to happen, but there are ray processes running in task manager.

What else can I try?

**Comment by freekmurze** [comment, marked as answer]
Hi @pbruelemans could you create an issue for this. Please fill in the issue template so we know your OS and version numbers. Thank you!

**Reply by pbruelemans** [reply]
Hi @freekmurze , I'm new to github.  Do I post it as a bug?  I'm not sure it's a bug, It could be something with my system.

**Reply by freekmurze** [reply]
Here's the direct link to create a bug report: https://github.com/spatie/ray/issues/new?assignees=&labels=&template=bug_report.md&title=
